.env.example

```diff
@@ -21,7 +21,7 @@ DGRAPH_ROOT_HASH=52a7ff82bd6f53b31285e97d6806d886eefb650e79754784e9d923d3df347c9
 
 GRAPHQL_URI=https://alpha-stts.tekit.io/graphql
 
-TELOS_ENDPOINTS=https://testnet.telos.caleos.io
+TELOS_ENDPOINTS=https://telos.caleos.io
```

Comment:
why endpont  was changed???

Reply:
I think is an error because I made a mistake merging master. I didn't see this change I'm going to restore that